ui/modal/create_password_modal.go

```diff
@@ -148,6 +159,11 @@ func (cm *CreatePasswordModal) Handle() {
 		}
 	}
 
+	if cm.modal.BackdropClicked() {
+		cm.Dismiss()
```

Comment:
A check should be made before closing to avoid closing the modal while a process is running.